traffic/configure-ios.md

```diff
@@ -42,15 +48,30 @@ To capture and inspect traffic on iOS devices, perform the following steps:
 
 1. Type the Fiddler Everywhere listening port (8866 by default) in the **Port** field. Tap **Save**.
 
-    >tip With the current setup you will be able to capture non-secure HTTP traffic. However, if you try to open any HTTPS website, you'll get the "This site's security certificate is not trusted!" error. To fix this issue, trust the Fiddler root certificate.
+    >tip With the current setup, you will be able to capture non-secure HTTP traffic. However, if you try to open any HTTPS website, you'll get the "This site's security certificate is not trusted!" error. To fix this issue, trust the Fiddler root certificate.
 
 1. Open a browser on the iOS device and type the http://ipv4.fiddler:8866 echo service address of Fiddler Everywhere.
 
 1. Click the Fiddler root certificate link to download it.
 
 1. On your iOS device, open **General** and install the certificate through **Profile Downloaded**. Note that this option will appear only after the certificate is downloaded.
 
-1. (For iOS 10.3+) Go to **General** > **About** > **Certificate Trust Settings** and enable full trust for the **DO_NOT_TRUST_FiddlerRoot** certificate. Note that you will see the **DO_NOT_TRUST_FiddlerRoot** certificate only after completing the previous step.
+1. (For iOS 10.3+) Go to **Settings** > **General** > **About** > **Certificate Trust Settings** and enable full trust for the **DO_NOT_TRUST_FiddlerRoot** certificate. Note that you will see the **DO_NOT_TRUST_FiddlerRoot** certificate only after completing the previous step.
+
+
+## Configure iOS Emulator
+
+The newer versions of the iOS emulators accessible through XCode are using the OS system proxy by default. However, due to [a security limitation](https://developer.apple.com/forums/thread/124056), the iOS emulator won't access the folders containing the trust certificates. To capture HTTPS traffic, you must manually export the Fiddler Everywhere certificate and install it on your iOS emulator.
```

Comment:
The later versions of the iOS emulators, which are accessible through XCode, use the OS system proxy by default. However, due to [a security limitation](https://developer.apple.com/forums/thread/124056), the iOS emulator won't access the folders containing the trust certificates. To capture HTTPS traffic, you must manually export the Fiddler Everywhere certificate and install it on your iOS emulator.